### PR TITLE
Added New pages

### LINKED ISSUE #8: Misc comments
- I like that the buttons and links on the landing page navigate to subpages inside docs. Maybe we can make this more consistent. That is, we map all buttons/links on the landing page to subpages . E.g. we could make "demo" and "friends"   subpages in docs, to which "Try the demo" and "Explore integrations" point to. Those pages can then contain a short description of demo and friends repos and pointers to those repos. I think that might be a less disruptive experience, compared to when the user is directed to external pages right away on the landing page.

- Why are there two search bars, one in the right top corner and one in the left navigation column? (no need to fix this now, I'm just curious)

- Regarding content I am not sure if we should follow the recommendation from the [cncf assessment](https://github.com/in-toto/in-toto.io/issues/35) and just re-publish the docs from python-in-toto as Getting started page. The documentation should represent the whole in-toto project, but the python-in-toto cli is only one of multiple in-toto implementations.

_Originally posted by @lukpueh in https://github.com/DarikshaAnsari/in-toto-docs/issues/7#issuecomment-2252204008_

### PR DESCRIPTION
**Pull request by DarikshaAnsari**
- Fixes #8

### CODE REVIEW

File: content/en/ecosystem/adopters.md
Comment (chalin) [inline content/en/ecosystem/adopters.md:1]: Replace this single file with:

- A directory named `adopters`
- Rename/move this file as `adopters/index.md` -- note the lack of underscore at the start
- Rename/move `static/images/img/Adopters_logo` to `adopters/img`
- Adjust the path in `adopters/index.md` accordingly

File: content/en/ecosystem/integrations.md
Comment (chalin) [inline content/en/ecosystem/integrations.md:1]: Make a change similar to the one described above for adopter.md. That is create a directory named `integrations`, etc.

File: content/en/ecosystem/adopters.md
Comment (chalin) [inline content/en/ecosystem/adopters.md]:
```suggestion
| Organization  |  | Description |
```

File: content/en/ecosystem/integrations.md
Comment (chalin) [inline content/en/ecosystem/integrations.md]:
```suggestion
| Organization |  | Description |
```

File: content/en/ecosystem/_index.md
Comment (chalin) [inline content/en/ecosystem/_index.md]:
```suggestion
description: Ecosystem of in-toto adopters and integrations.
```

File: content/en/ecosystem/adopters.md
Comment (chalin) [inline content/en/ecosystem/adopters.md]:
```suggestion
description: Projects and organizations using in-toto.
```

File: content/en/ecosystem/integrations.md
Comment (chalin) [inline content/en/ecosystem/integrations.md]:
```suggestion
description: Projects and products with in-toto integrations.
```

File: content/en/docs/demo.md
Comment (chalin) [inline content/en/docs/demo.md:1]: @DarikshaAnsari - I believe that you copied this from the README of the demo repo. We don't want the same information in two places, so either the README will need to be shortened, and a link added to this page (once it becomes a part of the official site), or we can find a way later to fetch the page a build time. 

I'll let @lukpueh comment on his preference here.

**Review by chalin — COMMENTED** [review]
A first round of comments.

Please run `npm run fix:format` and commit the changes.

File: content/en/ecosystem/adopters.md
Comment (chalin) [reply content/en/ecosystem/adopters.md:1]: This still needs to be done @DarikshaAnsari. For example:

- `content/en/ecosystem/adopters/_index.md` should be named `content/en/ecosystem/adopters/index.md`, without an underscore
- `static/images/adopters/img/` should be renamed to `content/en/ecosystem/adopters/img/`
- Paths to the images will need to be adjusted.

Do a similar change for the Integrations.

Comment (DarikshaAnsari) [reply content/en/ecosystem/adopters.md:1]: I didn't get your second point  `static/images/adopters/img/ `should be renamed to `content/en/ecosystem/adopters/img/` images are already in static folder does we have to shift it to `content/en/ecosystem/adopters/img`

Comment (chalin) [reply content/en/ecosystem/adopters.md:1]: > does we have to shift it to `content/en/ecosystem/adopters/img`

Yes, as you have done in the PR update. The images aren't rendering because the path is wrong. I've fixed the paths in a review comment below, see https://github.com/DarikshaAnsari/in-toto-docs/pull/11#discussion_r1712955328.

File: content/en/ecosystem/adopters/index.md
Comment (chalin) [inline content/en/ecosystem/adopters/index.md]:
```suggestion
| Datadog       |<img src="img/Datadog_logo.png" width="50" height="50">|Datadog uses in-toto to secure its agent integrations as they move through the company's CI/CD system. |
| OpenVEX       |<img src="img/OpenVEX_logo.png" width="50" height="50">|OpenVEX documents are designed to be self-sustaining, but the specification is designed to benefit from the in-toto attestation format completing VEX statements with data outside of the OpenVEX predicate. |
| SLSA          |<img src="img/SLSA_logo.svg" width="50" height="50">|Supply chain Levels for Software Artifacts, or SLSA, is a framework that provides a series of requirements and controls. |
| SolarWinds    |<img src="img/Solarwinds_Logo.png" width="50" height="50">|SolarWinds is an American company that provides information technology services and software to other companies and government agencies. |

```

File: content/en/ecosystem/integreations/index.md
Comment (chalin) [inline content/en/ecosystem/integreations/index.md]:
```suggestion
| GitLab           | <img src="img/Gitlab_logo.png" width="50" height="50">| GitLab is a popular Git server that also provides CI/CD integrations. |
| Grafeas          |<img src="img/Grafeas_logo.png" width="50" height="50">| Grafeas is an open source metadata API that is used to store metadata relevant to software supply chains. Grafeas includes support for in-toto link metadata. |
| GUAC             |<img src="img/Guac_logo.png" width="50" height="50">| GUAC has the ability to ingest and parse SLSA and other in-toto ITE6 attestations (either wrapped in DSSE or standalone). |
| Hoppr            || Hoppr leverages the in-toto python package to generate in-toto layout files based on a hoppr transfer configuration. |
| Jenkins          |<img src="img/Jenkins_logo.png" width="50" height="50">| The in-toto team maintains a plugin for Jenkins that can be used to generate in-toto metadata pertaining to a particular build or "job". |
| rebuilderd       || Rebuilderd is a build system project part of Reproducible Builds. When the result of a rebuild is positive, i.e., the build process is found to be reproducible, rebuilderd generates an in-toto link recording this result. |
| Sigstore         |<img src="img/Sistore_logo.png" width="50" height="50">| In-toto and Sigstore are complementary in their efforts, and Sigstore integrates in-toto in a number of ways. Sigstore's keyless signing can be used to sign in-toto metadata, as demonstrated by Cosign's SLSA Provenance generation. |
| Tekton Chains    | <img src="img/Tekton_logo.png" width="50" height="50">| Tekton Chains is a component for Tekton that adds software supply chain security. Chains observes all "TaskRuns" or jobs that are executed, and generates an in-toto attestation. |
| TestifySec       |<img src="img/Testifysec_logo.svg" width="50" height="50">| TestifySec is a software supply chain security company that has created two open source projects that leverage in-toto. Witness and Archivista. |

```

**Review by chalin — APPROVED** [review]
LGTM, once the paths to images are fixed in Adopters.

**Comment by DarikshaAnsari** [comment]
cc:- @lukpueh

File: content/en/_index.md
Comment (lukpueh) [inline content/en/_index.md]: This link is broken.

**Review by lukpueh — CHANGES_REQUESTED** [review]
Nice progress! Though it does not fully address #8, or at least not as requested. Did you disagree with my comments? Or was I just unclear. :) 

> * I like that the buttons and links on the landing page navigate to subpages inside docs. Maybe we can make this more consistent. That is, we map all buttons/links on the landing page to subpages . E.g. we could make "demo" and "friends"   subpages in docs, to which "Try the demo" and "Explore integrations" point to. Those pages can then contain a short description of demo and friends repos and pointers to those repos. I think that might be a less disruptive experience, compared to when the user is directed to external pages right away on the landing page.

By  **short description** I meant a paragraph or so and not the complete README from the demo and friends repos. Replicating all of this content on the website means that we have to either maintain it in two places, or remove it in its original place. Both of which does not seem desirable. 

> * Why are there two search bars, one in the right top corner and one in the left navigation column? (no need to fix this now, I'm just curious)

This does not seem resolved.

> * Regarding content I am not sure if we should follow the recommendation from the [cncf assessment](https://github.com/in-toto/in-toto.io/issues/35) and just re-publish the docs from python-in-toto as Getting started page. The documentation should represent the whole in-toto project, but the python-in-toto cli is only one of multiple in-toto implementations.

Same as above with demo and friends content.

**Comment by DarikshaAnsari** [comment]
> Nice progress! Though it does not fully address #8, or at least not as requested. Did you disagree with my comments? Or was I just unclear. :)
> 
> > * I like that the buttons and links on the landing page navigate to subpages inside docs. Maybe we can make this more consistent. That is, we map all buttons/links on the landing page to subpages . E.g. we could make "demo" and "friends"   subpages in docs, to which "Try the demo" and "Explore integrations" point to. Those pages can then contain a short description of demo and friends repos and pointers to those repos. I think that might be a less disruptive experience, compared to when the user is directed to external pages right away on the landing page.
> 
> By **short description** I meant a paragraph or so and not the complete README from the demo and friends repos. Replicating all of this content on the website means that we have to either maintain it in two places, or remove it in its original place. Both of which does not seem desirable.
> 
> > * Why are there two search bars, one in the right top corner and one in the left navigation column? (no need to fix this now, I'm just curious)
> 
> This does not seem resolved.
> 
> > * Regarding content I am not sure if we should follow the recommendation from the [cncf assessment](https://github.com/in-toto/in-toto.io/issues/35) and just re-publish the docs from python-in-toto as Getting started page. The documentation should represent the whole in-toto project, but the python-in-toto cli is only one of multiple in-toto implementations.
> 
> Same as above with demo and friends content.

Hey @lukpueh, you're right, but I believe the friends section looks nice on the website rather than residing in the README of any GitHub repository. Additionally, for the demo, it's beneficial for all the contributors and integrators of the project to have the demo on one website instead of having to jump to another link placed on GitHub. I think you get my point. I've taken reference from many CNCF documentation websites that do exactly the same, including OpenTelemetry,KCL and many more.

File: content/en/_index.md
Comment (lukpueh) [reply content/en/_index.md]: Ping

**Comment by lukpueh** [comment]
> > > * Why are there two search bars, one in the right top corner and one in the left navigation column? (no need to fix this now, I'm just curious)
> > 

If you don't want to fix this now, but close #8, please submit another issue so we can fix this separately.

**Comment by DarikshaAnsari** [comment]
> * Why are there two search bars, one in the right top corner and one in the left navigation column? (no need to fix this now, I'm just curious)

I think this is the layout of google docsy.

**Comment by lukpueh** [comment]
> Hey @lukpueh, you're right, but I believe the friends section looks nice on the website rather than residing in the README of any GitHub repository. Additionally, for the demo, it's beneficial for all the contributors and integrators of the project to have the demo on one website instead of having to jump to another link placed on GitHub. I think you get my point. I've taken reference from many CNCF documentation websites that do exactly the same, including OpenTelemetry,KCL and many more.

Okay, I'm still not happy of maintaining the exact same content in two places, but I won't insist. Let's have other in-toto project maintainers weigh in, when we merge this upstream.

**Review by lukpueh — APPROVED** [review]
You can fix the get started link in a separate PR.

File: content/en/docs/demo.md
Comment (chalin) [reply content/en/docs/demo.md:1]: I've created the following to track this issue, since @lukpueh and I agree that we **do not** want replicated content:

- #22

**Review by chalin — APPROVED** [review]
I'm going along with @lukpueh's approval. We can address the remaining issues via:

- #21 
- #22

**Comment by DarikshaAnsari** [comment]
@chalin @lukpueh I’m thinking of keeping the Adopters and Integrations page because it looks great on the website. Instead, I will update the README content for Friends on GitHub and link it to our website after deploying.

File: content/en/_index.md
Comment (chalin) [inline content/en/_index.md]: A link to the "Ecosystem" is in the top nav. We don't need a CTA button for this IMHO.

```suggestion
```

**Review by chalin — APPROVED** [review]
Please rebase and ensure that all tests pass. Other than the one suggestion (see the inline comment), this LGTM under the assumption that you'll be following up with further changes.

**Comment by DarikshaAnsari** [comment]
@chalin @lukpueh Can I merge this PR now.

**Comment by chalin** [comment]
> @chalin @lukpueh I’m thinking of keeping the Adopters and Integrations page because it looks great on the website. Instead, I will update the README content for Friends on GitHub and link it to our website after deploying.

👍🏻 will track this followup via #22.

**Review by chalin — APPROVED** [review]
Run `npm run fix:format` and commit changes; otherwise, this LGTM

**Comment by DarikshaAnsari** [comment]
I have used `npm run fix:format`  and after adding the code it is showing everything is uptodate.
```
PS C:\Users\asus\Documents\Opensource\myproject> git push origin patch-newpages
Everything up-to-date
```

**Comment by chalin** [comment]
> I have used `npm run fix:format` and after adding the code it is showing everything is uptodate.
> 
> ```
> PS C:\Users\asus\Documents\Opensource\myproject> git push origin patch-newpages
> Everything up-to-date
> ```

Right. You'll need to merge #27 first, then rebase this PR, resolve conflicts, re-run fix:format, and commit the results :).

**Pull request merged**
